.concourse/tasks/upgrade.sh

```diff
@@ -15,16 +15,19 @@ export FORCE_DELETE=true
 export HELM_VERSION="v3.1.1"
 export BACKEND=imported
 export QUIET_OUTPUT=true
-export GKE_CLUSTER_NAME="kubecf-ci-$(cat semver.gke-cluster/version | sed 's/\./-/g')"
-export KUBECFG="$(readlink -f ~/.kube/config)"
+GKE_CLUSTER_NAME="kubecf-ci-$(sed 's/\./-/g' < "semver.gke-cluster/version")"
+export GKE_CLUSTER_NAME
+KUBECFG="$(readlink -f ~/.kube/config)"
+export KUBECFG
 
-printf "%s" '((gke-suse-cap-json))' > $PWD/gke-key.json
+printf "%s" '((gke-suse-cap-json))' > "${PWD}/gke-key.json"
```

Comment:
I assume this is `printf` instead of `echo` to suppress printing EOL or something? It was like this before, so no further action needed, but still.

Reply:
not sure either but I spared myself one Concourse run by keeping it as it was. Lazy me.